py/server/deephaven/experimental/s3.py

```diff
@@ -45,9 +131,7 @@ def __init__(self,
                  num_concurrent_write_parts: Optional[int] = None,
                  profile_name: Optional[str] = None,
                  config_file_path: Optional[str] = None,
```

Comment:
Now that we have a `Credentials` object, is there any reason to also introduce a `Configuration` object and get rid of `config_file_path`?

Reply:
`config_file_path` is different from credentials. A config file contains a lot more stuff rather than just credentials, like region. That is why even though this file is used for credentials too, it's specified in instructions.